pkg/controller/controllercmd/builder.go

```diff
@@ -160,6 +183,11 @@ func (b *ControllerBuilder) WithLeaderElection(leaderElection configv1.LeaderEle
 	return b
 }
 
+func (b *ControllerBuilder) WithLeaderElectionTopologyDetector(topologyDetector TopologyDetector) *ControllerBuilder {
```

Comment:
```suggestion
func (b *ControllerBuilder) WithTopologyDetector(topologyDetector TopologyDetector) *ControllerBuilder {
```

This would be logically for everyone, correct?